mongo/client.go

```diff
@@ -180,9 +196,17 @@ func (c *Client) Disconnect(ctx context.Context) error {
 	return nil
 }
 
-// Ping verifies that the client can connect to the topology.
-// If readPreference is nil then will use the client's default read
-// preference.
+// Ping sends a ping command to verify that the client can connect to the cluster.
```

Comment:
Above you've used the word "deployment".  Here it's "cluster".  Let's pick one.

Reply:
Changed to "deployment" everywhere.